### PR TITLE
OCPBUG#14840: Modified note related to egress firewall policy in how DNS resolution works section

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.13+

Issue: [OCPBUGS 14840](https://issues.redhat.com/browse/OCPBUGS-14840)

Link to docs preview: [Doc Preview](https://68317--docspreview.netlify.app/openshift-enterprise/latest/networking/openshift_sdn/configuring-egress-firewall#domain-name-server-resolution_openshift-sdn-egress-firewall)

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

Additional information:

### CODE REVIEW

**Review by trozet — APPROVED** [review]
/lgtm

File: modules/nw-egressnetworkpolicy-about.adoc [hunk 1]
```diff
@@ -148,9 +148,9 @@ endif::ovn[]
 
 [NOTE]
 ====
-The egress firewall always allows pods access to the external interface of the node that the pod is on for DNS resolution.
+Using DNS names in your egress firewall policy does not affect local DNS resolution through CoreDNS.
```
Comment (dfitzmau) [inline modules/nw-egressnetworkpolicy-about.adoc:151]: Using / By using 

Screenshot from 2023-12-01 11-38-54

File: modules/nw-egressnetworkpolicy-about.adoc [hunk 2]
```diff
@@ -148,9 +148,9 @@ endif::ovn[]
...
 
-If you use domain names in your egress firewall policy and your DNS resolution is not handled by a DNS server on the local node, then you must add egress firewall rules that allow access to your DNS server's IP addresses. if you are using domain names in your pods.
+However, if your egress firewall policy uses domain names, and an external DNS server handles DNS resolution for an affected pod, you must include egress firewall rules permitting access to the IP addresses of your DNS server.
```
Comment (dfitzmau) [inline modules/nw-egressnetworkpolicy-about.adoc:153]:
```suggestion
However, if your egress firewall policy uses domain names, and an external DNS server handles DNS resolution for an affected pod, you must include egress firewall rules that permit access to the IP addresses of your DNS server.
```

**Review by mffiedler — APPROVED** [review]
/LGTM

**Comment by dfitzmau** [comment]
Hi @subhtk . Thanks for making the changes. Consider dropping the "However" as its redundant with the "If" clause.

LGTM

File: modules/nw-egressnetworkpolicy-about.adoc [hunk 3]
```diff
@@ -148,9 +148,9 @@ endif::ovn[]
 
 [NOTE]
 ====
-The egress firewall always allows pods access to the external interface of the node that the pod is on for DNS resolution.
+By using DNS names in your egress firewall policy does not affect local DNS resolution through CoreDNS.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/nw-egressnetworkpolicy-about.adoc:151]: Ack.

**Comment by skrthomas** [comment]
merge review lgtm

**Pull request merged**